docs/sources/helm-charts/mimir-distributed/migration-guides/migrate-to-kubernetes-version-1.25.md

```diff
@@ -32,3 +32,17 @@ Grafana Mimir does not require any special permissions on the hosts that it
 runs on. Because of this, you can deploy it in environments that enforce the
 Kubernetes [Restricted security policy](https://kubernetes.io/docs/concepts/security/pod-security-standards/).
 {{% /admonition %}}
+
+## Troubleshoot
+
+If you have upgraded to Kubernetes 1.25 and see the following error containing PodSecurityPolicy during a Helm release upgrade:
+
+```
+resource mapping not found for name: "mimir" namespace: "" from "":
+no matches for kind "PodSecurityPolicy" in version "policy/v1beta1" ensure CRDs are installed first
+```
+
```

Comment:
Should we add an explanation for why this is happening?
```suggestion
This happens because helm stores the current release in a Secret in the namespace. If the current release contains removed resources the `helm` command fails to determine the current state of the release.

To remove the PodSecurityPolicy from the Helm release history follow this procedure:
```